incubator/nodejs/README.md

```diff
@@ -1,56 +1,36 @@
 # Node.js Stack
 
-A stack is the foundation of your application. This stack provides a prototype platform definition for Node.js v10 based projects.
+The Node.js stack provides a consistent way of developing [Node.js](https://nodejs.org/) applications. As an asynchronous event-driven JavaScript runtime, Node is designed to build scalable network applications.
 
-## Templates
+This stack is based on `Node.js v10` runtime and allows you to bring any Node.js application that can work with this version of the runtime.
 
-A template is your starting ppoint to begin developing your application.
+## Templates
 
-### simple template
+Templates are used to create your local project and start your development. When initializing your project you will be provided with the simplest Node.js application you can possibly write.
```

Comment:
"you will be provided with the simplest Node.js application you can possibly write"
then we say "The default template provides a simple application that logs a message to the console"
I found this confusing as I wondered to start with if these were two different things

Why don't you merge sentences together, as I presume that simple Node.js app is created with the default template?

Reply:
addressed